messaginginapp/src/sharedTest/java/io/customer/messaginginapp/ModuleMessagingInAppTest.kt

```diff
@@ -31,6 +34,13 @@ internal class ModuleMessagingInAppTest : BaseTest() {
 
         module.initialize()
 
+        // verify gist is initialized
         verify(gistInAppMessagesProvider).initProvider(any(), eq("test"))
+
+        // verify hook was added
+        verify(hooksManager).add(any(), any())
```

Comment:
I think it would be good to verify the correct module was registered with the hooks manager. 

```suggestion
        verify(hooksManager).add(equal(.messagingInApp), any())
```